src/.formatted-files

```diff
@@ -0,0 +1,8 @@
+stat.h
+tweak.cpp
+tweak.h
+thinblock.cpp
+thinblock.h
+leakybucket.h
+parallel.cpp
+parallel.h
```

Comment:
Let say my PR is going to change something on `wallet.cpp` only.

Currently this travis task will be executed against my PR and will check the format correctness of unrelated files, e.g. `unlimited.cpp`,  right? 

This PR will introduce a check on format correctness for a list of files that is independent from the content of the PR. 

Hence such control will be useful only if the PR tat is going to be evaluated will change on the file listed in `src/.formatted-files`

I wonder if we could avoid to run the test if we are not touching such files. 

Take into account that the check requires a  fairly low amount of resources and that it is executed only for one configuration, this PR is good to go. We could improve it later on